bk-config/unit-tests/test_register.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+
+cd ./bk-config
+
+rm known_hosts register.pem register.pub register.pem-cert.pub
+
+
+set -e
+
+./create_client_files.sh bk-sshd 22 forever
+
+
+# TODO: The known_hosts file did not work here for some reason
+
+NEW_ID=$(docker run --rm  --network beekeeper_default -v `pwd`:/keys/:ro -w /keys -v beekeeper-config_bk-secrets:/usr/lib/sage/:ro sagecontinuum/bk-config bash -c 'apt-get install -y jq ; ssh -o StrictHostKeyChecking=no  sage_registration@bk-sshd -p 22 -i register.pem register 0000000000000001 | jq -r -j .id' | tail -n 1)
```

Comment:
do we want to define a network in the docker-compose ?

Reply:
Given that we may want to migrate this to k8s anyway, I do not think this is critical. But in general, you are right.